pkg/cli/gptscript.go

```diff
@@ -371,7 +371,7 @@ func (r *GPTScript) Run(cmd *cobra.Command, args []string) (retErr error) {
 				gptOpt.Env = append(gptOpt.Env, "GPTSCRIPT_SDKSERVER_CREDENTIAL_OVERRIDE="+strings.Join(r.CredentialOverride, ","))
 			}
 
-			args = append([]string{args[0]}, "--file="+file)
+			gptOpt.Env = append(gptOpt.Env, "GPTSCRIPT_CHAT_UI_RUN_FILE="+file)
```

Comment:
I would prefer not to use `GPTSCRIPT_` for environment variables that are not read/cared by `gptscript` itself.

Reply:
I feel like the environment variables used by the UI should at least have their own common prefix. Not sure what I would go with here though. Any recommendations?

Reply:
I went with `UI_RUN_FILE`. LMK what you think of that.